problems/variables.js

```diff
@@ -2,48 +2,78 @@
 
 // Age Calculator:
 //  * Store your birth year in a constant variable.
+    //var birthYear = 1999;
 //  * Store a future year in a variable.
+    //var futureYear = 2023;
 //  * Calculate your 2 possible ages for that year based on the stored values.
+    //var ageMax = futureYear - birthYear;
+    //var ageMin = futureYear - birthYear -1;
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
+
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
+    //console.log('I will be either ' + (ageMin) + ' or ' + (ageMax) + ' in ' + futureYear);    
 
 // ## Problem Two
 
 // Snack Supply Calculator:
 //  * Store your current age in a variable.
+    //var age = 20;
 //  * Store a maximum age in a constant variable.
+    //var maxAge = 30;
 //  * Store an estimated snack amount per day (as a number).
+    //var snackPerDay = 8;
 //  * Calculate how many snacks you would eat total, from your current age until the maximum age.
+    //var totalSnacks = (snackPerDay * 365) * (maxAge - age);
 //  * Log the result to the screen like this: "You will need `snackNumber` to last you until the age of `maxAge`".
-
+    //console.log (' You will need ' + (totalSnacks) + ' snacks ' + ' to last you until the age of ' + (maxAge));
 // ## Problem Three
 
 // Calculate properties of a circle, using the definitions: http://math2.org/math/geometry/circles.htm 
 //  * Store a radius into a variable.
+    //var radius = 6;
 //  * Calculate the circumference based on the radius, and log "The circumference is `circumferenceResult`".
+    //var circumferenceResult = Math.PI * 2*radius;
+    //console.log("The circumference is " + circumferenceResult);
 //  * Calculate the area based on the radius, and log "The area is `areaOfCircle`".
+    //var areaOfCircle = Math.PI * radius*radius;
```

Comment:
radius*radius works but look up Math.pow or other ways to square numbers in javascript!